front/components/assistant/AssistantDetails.tsx

```diff
@@ -130,13 +238,119 @@ export function AssistantDetails({
       variant="side-sm"
     >
       <div className="flex flex-col gap-5 pt-6 text-sm text-foreground">
-        <DescriptionSection />
-        <AssistantActionsSection
-          agentConfiguration={agentConfiguration}
-          owner={owner}
-        />
-        <InstructionsSection />
+        <TopSection />
+        <TabsSection />
       </div>
     </ElementModal>
   );
 }
+
+function ConfigVersionHeader({
+  agentConfigurationVersion,
+  agentConfiguration,
+  isLatestVersion,
+}: {
+  agentConfigurationVersion: number;
+  agentConfiguration: LightAgentConfigurationType | undefined;
+  isLatestVersion: boolean;
+}) {
+  const getStringRepresentation = useCallback(
+    (config: LightAgentConfigurationType) => {
+      const dateFormatter = new Intl.DateTimeFormat(navigator.language, {
+        year: "numeric",
+        month: "short",
+        day: "numeric",
+        hour: "numeric",
+        minute: "numeric",
+      });
+      return isLatestVersion
+        ? "Latest Version"
+        : config.versionCreatedAt
+          ? dateFormatter.format(new Date(config.versionCreatedAt))
+          : `v${config.version}`;
+    },
+    [isLatestVersion]
+  );
+
+  return (
+    <div className="flex items-center gap-2">
+      <Page.H variant="h6">
+        {agentConfiguration
+          ? getStringRepresentation(agentConfiguration)
+          : `v${agentConfigurationVersion}`}
+      </Page.H>
+    </div>
+  );
+}
+
+function FeedbackCard({
+  owner,
+  feedback,
+}: {
+  owner: LightWorkspaceType;
+  feedback: AgentMessageFeedbackType;
+}) {
+  const { userDetails } = useUserDetails(feedback.userId);
+  const { conversationId } = useFeedbackConversation({
+    workspaceId: owner.sId,
+    feedbackId: feedback.id.toString(),
+  });
+
+  const baseUrl =
+    process.env.NODE_ENV === "development"
+      ? "http://localhost:3000"
+      : "https://dust.tt";
+  const conversationUrl = `${baseUrl}/w/${owner.sId}/assistant/${conversationId}`;
+
+  return (
+    <ContentMessage variant="slate" className="my-2">
+      <div className="justify-content-around mb-3 flex items-center gap-2">
+        <div className="flex w-full items-center gap-2">
+          <Avatar
```

Comment:
Image will break if userDetails.image === null, display it conditionally 👍 (my fault haha)